mmv1/third_party/terraform/services/bigquery/resource_bigquery_table_test.go

```diff
@@ -4324,8 +4324,8 @@ resource "google_bigquery_table" "test" {
   dataset_id = "${google_bigquery_dataset.test.dataset_id}"
   table_id   = "%{table_id}"
   resource_tags = {
-    "%{project_id}/${google_tags_tag_key.key1.short_name}" = "${google_tags_tag_value.value1.short_name}"
-    "%{project_id}/${google_tags_tag_key.key2.short_name}" = "${google_tags_tag_value.value2.short_name}"
+    (google_tags_tag_key.key1.namespaced_name) = google_tags_tag_value.value1.short_name
```

Comment:
note: for these `namespaced_name` / ` short_name` pairs, I _suspect` they would work with `.id` on both side like in the other examples, but this makes the adjustments more minimal, as I believe it should be exactly equivalent with the old value.

If someone has a strong opinion about which is correct, they could easily be updated.